seed/seed20.go

```diff
@@ -448,9 +640,42 @@ func (s *seed20) lookupSnap(snapRef naming.SnapRef, optSnap *internal.Snap20, ch
 		timings.Run(tm, "derive-side-info", fmt.Sprintf("hash and derive side info for snap %q", snapRef.SnapName()), func(nested timings.Measurer) {
```

Comment:
Maybe worth creating a `deriveSideInfo` function/method that can return an error, making it so we only have to do the dance of making sure we don't shadow the `err` variable once?

Reply:
Done now